app/uploaders/simple_forms_api/form_remediation/uploader.rb

```diff
@@ -23,6 +23,14 @@ def get_s3_link(file_path)
                          .object(file_path)
                          .presigned_url(:get, expires_in: 30.minutes.to_i)
         end
+
+        def get_s3_file(from_path, to_path)
+          new_s3_resource.bucket(s3_settings.bucket)
+                         .object(from_path)
+                         .get(response_target: to_path)
+        rescue Aws::S3::Errors::NoSuchKey
+          nil
```

Comment:
do we want to do something on error?
do we also want a generic error?

Reply:
This error should only be thrown when the directory doesn't exist. In that case, we'll just create a new one.

Yeah, good idea with the generic error, I had a rescue in there for one but I think moving it around and refactoring killed it at some point.